.golangci.yml

```diff
@@ -21,7 +21,9 @@ linters:
     - staticcheck
     - ineffassign
     - unused
-issues: 
+issues:
+  exclude:
+    - composites
```

Comment:
the examples project was having a type error, it was fixed by using `github.Timestamp`, which caused another error in `go vet` relating to unkeyed fields. excluding `composites` fixes the error

Reply:
Curious what we were missing by excluding composites, I ran across these issues:

* https://github.com/golangci/golangci-lint/issues/446
* https://github.com/golang/go/issues/43864

in case anyone in the future finds this and wonders about it.